packages/design-system/src/components/landingPage/cardsPanel.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies.
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies.
+ */
+import { SIDEBAR_ITEMS_KEYS, useSidebar } from '../sidebar';
+
+type PinnedItem = {
+  name: string;
+  icon: React.ComponentType<{
+    width: number;
+    height: number;
+    className?: string;
+  }>;
+  sidebarKey: SIDEBAR_ITEMS_KEYS;
+};
+
+type FeatureItem = PinnedItem & {
+  description: string;
+  colorClasses?: {
+    heading?: string;
+  };
+  buttons?: {
+    name: string;
+    sidebarKey: SIDEBAR_ITEMS_KEYS;
+  }[];
+};
+
+type Props = {
```

Comment:
```suggestion
type CardsPanelProps = {
```

for the sake of consistency with all other components.

Reply:
Odd, I think I think I used the autocomplete here, I should have noticed when self reviewieng though, thanks